pdata/README.md

```diff
@@ -0,0 +1,144 @@
+# Pipeline data (pdata)
```

Comment:
Add one section about the "flags". I think we had a good conversation with conclusions there as well.

Reply:
Added sections for enums and flags